README.md

```diff
@@ -5,6 +5,7 @@ Heroku Homebrew Tap
 
 Install the Heroku CLI with
 
+    $ brew tap heroku/brew
```

Comment:
Tapping is automatic if you install with the fully qualified formula name (as is done here with `brew install heroku/brew/heroku`